src/dune_config/dune

```diff
@@ -0,0 +1,5 @@
+(library
+ (name dune_config)
+ ;; careful with adding dependencies here
+ ;; anything that depends on this library cannot itself be configurable
```

Comment:
I don't understand this comment, considering the one in the line above. Shouldn't it say "anything that this library depends on"?